docs/example-scenario/infrastructure/regulated-multitier-app.md

```diff
@@ -6,7 +6,7 @@ ms.date: 07/11/2018
 ---
 # Secure Windows web application for regulated industries
 
-This sample scenario is applicable to regulated industries that have a need to secure multi-tier applications. In this scenario, a front-end ASP.NET application securely connects to a protected back-end Microsoft SQL Server cluster.
+This sample scenario is applicable to regulated industries that have a need to secure multi-tier applications. In this scenario, a frontend ASP.NET application securely connects to a protected backend Microsoft SQL Server cluster.
```

Comment:
I think front-end and back-end should remain hyphenated when used as adjectives.

Reply:
yes, that is correct, reverted.